src/styles/blocks/contacts.scss

```
@@ -0,0 +1,123 @@
+.contacts {
+  &__container {
+    @extend %container;
+
+    @include mediamin(768px) {
+      display: flex;
+      align-items: flex-start;
+      flex-wrap: wrap;
+    }
+  }
+
+  &__heading {
+    margin-bottom: 48px;
+
+    text-align: center;
+
+    @include mediamin(768px) {
+      flex: 0 0 100%;
+    }
+  }
+
+  &__form {
+    display: flex;
+    flex-direction: column;
+
+    margin-bottom: 40px;
+
+    @include mediamin(768px) {
+      flex: 0 0 calc(50% - 12px);
+
+      margin-bottom: 0;
+      margin-right: 24px;
+    }
+  }
+
+  &__field {
+    padding: 14px 16px;
+
+    background-color: $color-gray;
+    border-radius: 40px;
+    border: 1px solid $color-gray;
+    outline: none;
+
+    color: $color-dark;
+    font-weight: 400;
+    font-size: 14px;
+    line-height: 143%;
+
+    transition: border$transition-props;
+
+    &::placeholder {
+      color: $color-jumbo;
+      font-weight: 400;
+      font-size: 14px;
+      line-height: 143%;
+    }
+
+    &:hover {
+      border: 1px solid $color-athens;
+    }
+
+    &:focus {
+      border: 1px solid $color-dark;
```

you r changing border color only in this rules, so you can use `border-color` instead of full border rule)

```suggestion
      border-color:  $color-athens;
    }

    &:focus {
      border-color: $color-dark;
```